static/js/subs.js

```diff
@@ -322,9 +322,9 @@ function actually_filter_streams() {
         subscribed_only = false;
     }
     exports.filter_table({ input: query, subscribed_only: subscribed_only });
-}
+};
 
-var filter_streams = _.throttle(actually_filter_streams, 50);
+var filter_streams = _.throttle(exports.actually_filter_streams, 50);
```

Comment:
Why not export `filter_streams` instead?

Reply:
I'd find that more confusing when reviewing code if there's `filter_streams`, but it's not the same as `exports.filter_streams`.

Reply:
Sorry, the question was why didn't you export the existing `filter_streams` instead of `actually_filter_streams`, not why did you pick the name :).

Reply:
(Or another way to put it, do we want this to not be throttled?)